solarforecastarbiter/reports/figures/plotly_figures.py

```diff
@@ -364,6 +365,63 @@ def scatter(timeseries_value_df, timeseries_meta_df, units):
     return fig
 
 
+def event_histogram(timeseries_value_df, timeseries_meta_df, units):
```

Comment:
`units` does not appear to be used or documented. Based on the way this function is called, it looks to me like the parameter should just be removed. Maybe there is an argument for keeping it for api consistency, but then it should still be documented as unused.

Reply:
I'll just remove `units` from `event_histogram()`.